x-pack/filebeat/module/ibmmq/errorlog/_meta/fields.yml

```diff
@@ -0,0 +1,80 @@
+- name: errorlog
+  description: IBM MQ error logs
+  example: Please add example
+  type: group
+  fields:
+  - name: process
+    description: Please add description
+    example: Please add example
+    type: text
+  - name: user
+    description: Please add description
+    example: Please add example
+    type: keyword
+  - name: program
+    description: Please add description
+    example: Please add example
+    type: keyword
+  - name: hostname
+    description: Please add description
+    example: Please add example
+    type: text
+  - name: installation
+    description: Please add description
+    example: Please add example
+    type: keyword
+  - name: vrmf
+    description: Please add description
+    example: Please add example
+    type: text
+  - name: qmgr
+    description: Please add description
+    example: Please add example
+    type: text
+  - name: iso8601_timestamp
+    description: Please add description
+    example: Please add example
+  - name: remotehost
+    description: Please add description
+    example: Please add example
+    type: text
+  - name: arithinsert1
```

Comment:
The meaning is different based on the error code. Is there any way to reflect this? Same for comment fields.

Reply:
Well its possibly one per error code. And I didn't found a documentation for that on IBM side. 
So weit can ignore that fact like we do in Winlogbeat.
We can talk to IBM directly and see weather they can share more insights.
Or we can try to build our own documentation. Which would be hard I guess.
I will see what I can get from IBM and otherwise I think we need to stay with the first option. This fields mostly do not add relevant info as the values are part of the message anyways.

Reply:
What about merging all artinsert into 1 field and the commentinsert into 1 field. Each field in Elasticsearch is an array, so the values would look like `arthinsert: ["val1", "val2"]`. Same for commentinsert. The main difference is that afterwards no query can be made "only" on arthinsert2. Would that be an issue?

Reply:
Well so far I have seen only one case where these information might be interesting. So I think its okay to do it like you suggested it.

Reply:
I put all together for now.